<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorDialog</class>
 <widget class="QDialog" name="ItemEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="2" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMain">
      <attribute name="title">
       <string>&amp;Основная информация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="1" column="2" colspan="2">
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Наименование</string>
         </property>
         <property name="buddy">
          <cstring>edtName</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLineEdit" name="edtCode"/>
       </item>
       <item row="12" column="2" colspan="2">
        <widget class="QCheckBox" name="chkCitizenship">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="lblCitizenship">
         <property name="text">
          <string>Гражданство Р&amp;Ф</string>
         </property>
         <property name="buddy">
          <cstring>chkCitizenship</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="lblLeftPartRegExp">
         <property name="text">
          <string>&amp;Левая часть серии</string>
         </property>
         <property name="buddy">
          <cstring>edtLeftPartRegExp</cstring>
         </property>
        </widget>
       </item>
       <item row="11" column="3">
        <widget class="QPushButton" name="btnTestNumber">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Проверить</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QCheckBox" name="chkHasRightPart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="CRBComboBox" name="cmbGroup"/>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="edtLeftPartRegExp">
         <property name="maxLength">
          <number>128</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Серия из двух &amp;частей</string>
         </property>
         <property name="buddy">
          <cstring>chkHasRightPart</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QLineEdit" name="edtFederalCode">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="13" column="1" colspan="2">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>103</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="2" colspan="2">
        <widget class="QLineEdit" name="edtPartSeparator">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maxLength">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QPushButton" name="btnTestRightPart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Проверить</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lblRightPartRegExp">
         <property name="text">
          <string>&amp;Правая часть серии</string>
         </property>
         <property name="buddy">
          <cstring>edtRightPartRegExp</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="lblRegionalCode">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Территориальный код</string>
         </property>
         <property name="buddy">
          <cstring>edtRegionalCode</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QLabel" name="lblPartSeparator">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Разделитель</string>
         </property>
         <property name="buddy">
          <cstring>edtPartSeparator</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QPushButton" name="btnTestLeftPart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Проверить</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QLineEdit" name="edtRightPartRegExp">
         <property name="maxLength">
          <number>128</number>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QLineEdit" name="edtNumberRegExp">
         <property name="maxLength">
          <number>128</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="lblFinance">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Группа документов</string>
         </property>
         <property name="buddy">
          <cstring>cmbGroup</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="QLineEdit" name="edtRegionalCode">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="lblNumberRegExp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Но&amp;мер</string>
         </property>
         <property name="buddy">
          <cstring>edtNumberRegExp</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="lblFederalCode">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Федеральный код</string>
         </property>
         <property name="buddy">
          <cstring>edtFederalCode</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblCode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Код</string>
         </property>
         <property name="buddy">
          <cstring>edtCode</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblTitle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Наименование для печати</string>
         </property>
         <property name="buddy">
          <cstring>edtTitle</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QLineEdit" name="edtTitle"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabIdentification">
      <attribute name="title">
       <string>&amp;Идентификация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="CInDocTableView" name="tblIdentification"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtFederalCode</tabstop>
  <tabstop>edtRegionalCode</tabstop>
  <tabstop>cmbGroup</tabstop>
  <tabstop>edtLeftPartRegExp</tabstop>
  <tabstop>btnTestLeftPart</tabstop>
  <tabstop>chkHasRightPart</tabstop>
  <tabstop>edtPartSeparator</tabstop>
  <tabstop>edtRightPartRegExp</tabstop>
  <tabstop>btnTestRightPart</tabstop>
  <tabstop>edtNumberRegExp</tabstop>
  <tabstop>btnTestNumber</tabstop>
  <tabstop>chkCitizenship</tabstop>
  <tabstop>tblIdentification</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkHasRightPart</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtRightPartRegExp</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkHasRightPart</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnTestRightPart</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkHasRightPart</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblPartSeparator</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkHasRightPart</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtPartSeparator</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkHasRightPart</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblRightPartRegExp</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
